### DISCUSSION TITLE
ZOHD Drift crashes when flown over 60 km/h, nav_manual_speed seems not working for wing

### DISCUSSION THREAD

**Discussion by deethee** [iNavFlight/inav#7552] [Q&A] [answered]
ZOHD Drift crashes when flown over 60 km/h

set nav_manual_speed = 500
This is only 18 km/h
It seems to be inactive for planes.

Limit via throttle will not work.
Is there any other parameter to set to limit plane speed everywhere?

**Comment by MrD-RC** [comment, marked as answer]
[nav_manual_speed](https://github.com/iNavFlight/inav/blob/master/docs/Settings.md#nav_manual_speed)  is for multirotors

The simplest way to limit the planes speed is to lower the throttle stick. Navigation modes have cruise throttle, which should be set for the plane.

**Reply by deethee** [reply]
does this work if no mode or manual mode is set? It doesn't help with diving either.

**Reply by MrD-RC** [reply]
If you’re in manual or acro, you’re **flying** the plane. So you control the throttle. If you’re in manual or acro and in a dive, you should be checking for overspeeds. Reduce the throttle with the gimbal. If you’re in a dive and still too fast, pull up slowly. 

The most I think should be an alarm to flash the speed indicators. Maintaining the correct speed for the airframe is the pilot’s responsibility, especially in manual.

**Comment by xxJohnxx** [comment]
I definetly agree with MrD-RC. There should be no throttle/pitch inputs to prevent overspeeds in manual or acro mode - this could result in very unpredictable behaviour if there is something wrong with one of the sensors.

Regarding the problem with the ZOHD Drift: Try adding some duct-tape to the leading edge of the wings, especially in the area around the wing tips. The Drift has issues with wing warp at high speeds, causing aileron reversal. I modified mine with a bit of tape, and I have never had any issues regarding the flight characteristics at higher speeds.
